src/documentation/sections/overview-section.ts

```diff
@@ -21,7 +21,7 @@ export const buildOverviewSection = ({ config }: { config: ProjectConfig }): str
 
 **Goal:** ${projectGoal}
 
-**Framework:** ${framework === 'agno' ? 'Agno' : 'Mastra'}
+    **Framework:** ${framework === 'agno' ? 'Agno' : framework === 'langgraph-py' ? 'LangGraph (Python)' : framework === 'langgraph-ts' ? 'LangGraph (TypeScript)' : 'Mastra'}
```

Comment:
nit: move this out into a function with a switch statement as chained terniaries become hell